### PR TITLE
Feature/all results

### PR DESCRIPTION
**Pull request by HryshkoAnton**
Clicking on 'see all results' button, user will be able to see all results.

https://github.com/ita-social-projects/GreenCity/issues/855

prev result:
image

new result:
image

### CODE REVIEW

File: src/app/component/layout/components/search-all-results/item/item.component.scss [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    mix-blend-mode: normal;
+    opacity: 0.6;
+  }
+
+  .icon {
+    align-items: center;
+    margin-right: 4px;
+    margin-top: -3px;
+  }
+}
```
Comment (yulkats) [inline src/app/component/layout/components/search-all-results/item/item.component.scss:74]: Don`t use negative values

File: src/app/component/layout/components/search-all-results/item/item.component.scss [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    color: #666;
+    mix-blend-mode: normal;
+    opacity: 0.6;
+  }
+
+  .icon {
+    align-items: center;
+    margin-right: 4px;
+    margin-top: -3px;
+  }
```
Comment (yulkats) [inline src/app/component/layout/components/search-all-results/item/item.component.scss:143]: Same as line 72

File: src/app/component/layout/components/search-all-results/item/item.component.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+@Component({
+  selector: 'app-item',
+  templateUrl: './item.component.html',
+  styleUrls: ['./item.component.scss']
+})
+export class ItemComponent {
+
+  @Input() searchModel;
+  profileIcons = ecoNewsIcons;
+
```
Comment (yulkats) [inline src/app/component/layout/components/search-all-results/item/item.component.ts:13]: remove these empty line

File: src/app/component/layout/components/search-all-results/search-all-results.component.ts [hunk 4]
```diff
@@ -12,7 +12,7 @@ import { NewsSearchModel } from '@global-models/search/newsSearch.model';
 export class SearchAllResultsComponent implements OnInit, OnDestroy {
   public inputValues = ['relevance', 'newest', 'latest'];
   public displayedElements: NewsSearchModel[] = [];
-  public elements: NewsSearchModel[];
+  public elements;
```
Comment (yulkats) [inline src/app/component/layout/components/search-all-results/search-all-results.component.ts:15]: add new model

File: src/app/component/layout/components/search-popup/search-popup.component.ts [hunk 5]
```diff
@@ -51,6 +52,11 @@ export class SearchPopupComponent implements OnInit, OnDestroy {
     }
   }
 
+  public getAllResults(category): void {
+    this.search.getAllResults(this.inputValue, category);
+    this.closeSearch();
+  }
```
Comment (yulkats) [inline src/app/component/layout/components/search-popup/search-popup.component.ts:58]: add argument type.

File: src/app/component/layout/components/search-all-results/item/item.component.scss [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+    font-size: 0.9rem;
+    color: #313131;
+    display: inline-block;
+    padding: 0.3rem 1rem;
+    border: 1px solid #313131;
+    border-radius: 53px;
+    margin-right: 1rem;
+    margin-bottom: 0.3rem;
+  }
+
```
[the remark this answers is not in the document]
Comment (HryshkoAnton) [reply src/app/component/layout/components/search-all-results/item/item.component.scss:25]: fixed

File: src/app/component/layout/components/search-all-results/item/item.component.scss (hunk 1 above)
Comment (HryshkoAnton) [reply src/app/component/layout/components/search-all-results/item/item.component.scss:73]: fixed

File: src/app/component/layout/components/search-all-results/item/item.component.scss (hunk 2 above)
Comment (HryshkoAnton) [reply src/app/component/layout/components/search-all-results/item/item.component.scss:142]: fixed

File: src/app/component/layout/components/search-all-results/item/item.component.ts (hunk 3 above)
Comment (HryshkoAnton) [reply src/app/component/layout/components/search-all-results/item/item.component.ts]: fixed

File: src/app/component/layout/components/search-all-results/search-all-results.component.ts (hunk 4 above)
Comment (HryshkoAnton) [reply src/app/component/layout/components/search-all-results/search-all-results.component.ts]: fixed

File: src/app/component/layout/components/search-popup/search-popup.component.ts (hunk 5 above)
Comment (HryshkoAnton) [reply src/app/component/layout/components/search-popup/search-popup.component.ts:58]: fixed

**Review by yulkats — APPROVED** [review]

**Review by mr-pavlenko — APPROVED** [review]

**Pull request merged**